tests/runtime_test/heterogeneousConfigs.yaml

```diff
@@ -0,0 +1,15 @@
+---
+- name:     Device1
+  backendName: CPU
+  parameters: |
+    "nonSupportedNodes":"AvgPool"
```

Comment:
Can you add `supportedNodes` example as well, because it needs to be tested?

Reply:
The "Device3" in this file has the "supportedNodes" params. Since there are many ops in resnet50, I just let Interpreter supports all. In unit test, we tested the supported nodes set.